src/wrap-store/wrapStore.js

```diff
@@ -47,8 +47,18 @@ export default (store, {
       const action = Object.assign({}, request.payload, {
         _sender: sender
       });
-      dispatchResponder(store.dispatch(action), sendResponse);
-      return true;
+
+      let dispatchResult = null;
+
+      try {
+        dispatchResult = store.dispatch(action);
+      } catch (e) {
+        dispatchResult = Promise.reject(e.message);
+        console.error(e);
+      } finally {
```

Comment:
Gonna need to test this out. I don't quite remember if not having the finally block would break the response dispatch.